### PR TITLE
apfree-wifidog: update to 7.10.2082

### PR DESCRIPTION
**Pull request by liudf0716**
Maintainer: @liudf0716
Compile tested: x86_64
Run tested: x86_64

Description:
   
https://github.com/liudf0716/apfree-wifidog/releases/tag/7.10.2082

Dengfeng Liu <liudf0716@gmail.com>

### CODE REVIEW

File: net/apfree-wifidog/files/wifidogx.init [hunk 1]
```diff
@@ -2,160 +2,181 @@
...
-		uci set wifidogx.$section.gateway_id=$gateway_id
-		uci commit wifidogx
-	fi
-
-	echo "GatewaySetting { 
-	GatewayInterface $gateway_name
-	GatewayChannel $gateway_channel
-	GatewayID $gateway_id
-}" >> ${CONFIGFILE}
+    local section="$1"
```
Comment (1715173329) [inline net/apfree-wifidog/files/wifidogx.init:11]: Please use tabs for indentation as always.

**Comment by 1715173329** [comment]
so what you did is clearly more than just "support local auth".
please describe your changes correctly, and would be better - split them, e.g. one commit for adding local auth parameter, one commit for shellcheck, one commit for ...

please also do not mix irrelevant changes, e.g. the indentation fix should be done in your first commit.

**Comment by liudf0716** [comment]
> so what you did is clearly more than just "support local auth". please describe your changes correctly, and would be better - split them, e.g. one commit for adding local auth parameter, one commit for shellcheck, one commit for ...
> 
> please also do not mix irrelevant changes, e.g. the indentation fix should be done in your first commit.

@1715173329 
For the init and conf files, two changes have been made:

Refactoring – The init file was restructured to improve code organization and maintainability.
Local Authentication Support – Added support for user authentication locally, allowing users to authenticate without relying on a cloud server.

File: net/apfree-wifidog/Makefile [hunk 2]
```diff
@@ -27,7 +27,7 @@ define Package/apfree-wifidog
   SUBMENU:=Captive Portals
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+zlib +libjson-c +libevent2 +libevent2-openssl +libuci +openssl-util +libnetfilter-queue
+  DEPENDS:=+zlib +libjson-c +libevent2 +libevent2-openssl +libuci +openssl-util +libnetfilter-queue +conntrack +libmosquitto
```
Comment (1715173329) [inline net/apfree-wifidog/Makefile:30]: I guess these are part of the update, but please mention them in the commit message.

File: net/apfree-wifidog/Makefile [hunk 3]
```diff
@@ -60,6 +60,7 @@ define Package/apfree-wifidog/install
 	$(INSTALL_BIN) ./files/wifidogx.init $(1)/etc/init.d/wifidogx
 	$(INSTALL_DIR) $(1)/etc/config
 	$(CP) ./files/wifidogx.conf $(1)/etc/config/wifidogx
+	$(INSTALL_DIR) $(1)/etc/wifidogx
```
Comment (1715173329) [inline net/apfree-wifidog/Makefile:63]: Same here ;)

Comment (liudf0716) [reply net/apfree-wifidog/Makefile:63]: @1715173329 

I apologize for any confusion caused by the new directory '/etc/wifidogx.' This directory is intended to store the local portal page for local authentication when it is enabled. However, maybe it is not the appropriate location for holding the local portal page.

File: net/apfree-wifidog/Makefile (hunk 2 above)
Comment (liudf0716) [reply net/apfree-wifidog/Makefile:30]: @1715173329 
Conntrack was utilized to fix a bug in Apfree-Wifidog, where a user's long-lived connection persisted even after logging out.
Libmosquitto was integrated as a preparatory step toward future MQTT protocol support, though the implementation is not yet complete.
For more detailed information, please refer to the release notes included in this PR.
https://github.com/liudf0716/apfree-wifidog/releases/tag/7.10.2082

Thank you for the thorough code review!

Comment (1715173329) [reply net/apfree-wifidog/Makefile:30]: Please mention these changes in commit message, thanks.

**Comment by liudf0716** [comment]
@1715173329 thanks

**Pull request merged**